daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -1131,6 +1131,13 @@ class RazerDeathStalkerExpert(_MacroKeyboard):
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/49/49_razer_deathstalker.png"
 
 
+class RazerDeathStalkerEssential(RazerDeathStalkerExpert):
```

Comment:
Please just duplicate the content of `RazerDeathStalkerExpert` class and don't inherit since those are just two different devices that happen to share the same feature set